### ISSUE TITLE
Some Nova translations break in combination with nova-flexible-content button translation

### ISSUE THREAD

**Issue by novelnet** [whitecube/nova-flexible-content#350]
I have set the locale to 'de'.
When using nova-flexbile-content anywhere in Nova the translations break in some places only, like here:

![image](https://user-images.githubusercontent.com/31067676/167263401-0822809e-3e57-4c45-b105-c0c208b6a085.png)

When i remove the translation function  `__('Add layout')` in Flexible.php everything works in Nova again:
```
public function __construct($name, $attribute = null, $resolveCallback = null)
    {
        parent::__construct($name, $attribute, $resolveCallback);

        $this->button('Add layout');
        ...
```
![image](https://user-images.githubusercontent.com/31067676/167263470-79541e98-4bf2-4410-bdbf-d0dfaf654a12.png)

This issue has multiple users probably and there is also one open at Nova directly: laravel/nova-issues#4192

Thanks in advance.

**Comment by novelnet** [comment]
Can be easily solved by translating `{{ __(field.button) }}` in <default-button /> of SearchMenu.vue and OriginalDropMenu.vue, instead of the Field Constructor (before the translations are loaded yet). I can provide a pull request

**Comment by ianrobertsFF** [comment]
I'd make a pull request if possible, at least then the maintainers can just review and merge when they can.

**Comment by novelnet** [comment]
Yes, i misunderstood that. In the meantime i ended up in overriding the constructor for myself, removing the translation function and using custom labeling:
```
class FlexibleOverride extends Flexible {
    public function __construct($name, $attribute = null, $resolveCallback = null)
    {
        \Laravel\Nova\Fields\Field::__construct($name, $attribute, $resolveCallback);

        #https://github.com/whitecube/nova-flexible-content/issues/350
        $this->button('Add layout');

        // The original menu as default
        $this->menu('flexible-drop-menu');

        $this->hideFromIndex();
    }
}
```
May dig deeper in looking for a better solution to commit, when i have some time off.

**Comment by voidgraphics** [comment]
Thanks for investigating this. Would definitely appreciate a PR with a fix. 👍
